tuf-spec.md

```diff
@@ -688,9 +688,11 @@ The "signed" portion of <a>root.json</a> is as follows:
 
   : <dfn>CONSISTENT_SNAPSHOT</dfn>
   ::
-    A boolean indicating whether the repository supports
-    consistent snapshots.  Section 7 goes into more detail on the consequences
-    of enabling this setting on a repository.
+    An optional boolean indicating whether the repository supports
+    consistent snapshots. This field is optional for backward compatibility with
+    old metadata that may exist without it. New implementations SHOULD include
+    it. Section 7 goes into more detail on the consequences of enabling
+    this setting on a repository.
```

Comment:
Let's both ensure we are cross-linking and avoid referring to sections by their explicit numbering
```suggestion
    it. [[consistent-snapshots]] goes into more detail on the consequences of enabling
    this setting on a repository.
```